vcpkg/triplets/arm-neon-android.cmake

```diff
@@ -1,6 +1,9 @@
 set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE static)
+set(VCPKG_CRT_LINKAGE dynamic)

```

Comment:
I think you could kill this neon triplet file altogether btw

Reply:
Not at all, r25 drops non neon, we should drop the current arm apks instead. Neon is faster and supported by virtually any device.